index.js

```diff
@@ -9,11 +17,170 @@ wrapId.addEventListener('click', () => {
+    featureimage: './images/popupimage.png',
+    technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+    linktoliveversion: '#',
+    linktosource: '#',
+  }, {
+    id: 'card7',
+    Name: 'Profesional Art Printing Data',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+    featureimage: './images/popupimage.png',
+    technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+    linktoliveversion: '#',
+    linktosource: '#',
+  }];
+
+const card1 = document.querySelector('#card0');
+const card2 = document.querySelector('#card1');
+const card3 = document.querySelector('#card2');
+const card4 = document.querySelector('#card3');
+const card5 = document.querySelector('#card4');
+const card6 = document.querySelector('#card5');
+const card7 = document.querySelector('#card6');
+
+const cardArray = [card1, card2, card3, card4, card5, card6, card7];
+
+for (let i = 0; i < cardArray.length; i++) {
+  cardArray[i].addEventListener('click', () => {
+    portfolio.classList.toggle('active');
+    navbar.classList.toggle('active');
+    headlinesection.classList.toggle('active');
+    aboutme.classList.toggle('active');
+    contactsection.classList.toggle('active');
+    footercontainer.classList.toggle('active');
+    // eslint-disable-next-line no-unused-vars
+    popupArray.forEach((items, index) => {
+      const container = document.querySelector('.container');
+      const section = document.createElement('section');
+      container.appendChild(section);
+      section.innerHTML = `<div class="pop-up-container">
+      <header class="pop-up-header">
+        <div class="pop-up-heading">
+          <h2 id="pop-up-heading">${items.Name}</h2>
+          <button class="close-btn">&times;</button> 
+        </div>
+        <div class="pop-up-buttons">
+          <span class="pop-up-lang-btn">${items.technologies[0]}</span>
+          <span class="pop-up-lang-btn">${items.technologies[1]}</span>
+          <span class="pop-up-lang-btn">${items.technologies[2]}</span>
+        </div>
+      </header>
+      <section class="pop-up-body">
+        <div class="pop-up-image">
+          <img src="${items.featureimage}" alt="Projectimg">
+        </div>
+        <div class="pop-up-details">
+          <p>${items.description}</p>
+          <div class="pop-up-details-btn-container">
+            <a href="${items.linktoliveversion}"><button class="pop-up-details-btn">See Live <span><img src="./images/icon-see-live.png" class='icon-see-live-img' alt="iconseeimage"></span></button></a>
```

Comment:
- [ ] Your image is still broken and the reason why is this happening is because you have a typo on your src instead of `src="./images/icon-see-live.png` it needs to be with the capital I `src="./images/Icon-see-live.png`
![image](https://user-images.githubusercontent.com/62705947/216373779-bc13649c-a715-49ed-a387-e8a1644b0e38.png)
![image](https://user-images.githubusercontent.com/62705947/216373949-8ba8cabf-1718-4108-bd3e-422853ae4749.png)